plugins/services/src/js/reducers/__tests__/JSONMultiContainer-test.js

```diff
@@ -0,0 +1,84 @@
+const Batch = require("#SRC/js/structs/Batch");
+const { combineParsers } = require("#SRC/js/utils/ParserUtil");
+const { combineReducers } = require("#SRC/js/utils/ReducerUtil");
+const JSONMultiContainerParser = require("../JSONMultiContainerParser");
+const JSONMultiContainerReducers = require("../JSONMultiContainerReducers");
+
+describe("JSONMultiContainer", function() {
```

Comment:
I'd rather not have it as a part of `JSONMultiContainerParser` because then we should check transactions and my goal was to check how Parser and Reducer work together and if JSON stays intact. 

We could have a separate test for `JSONMultiContainerParser` though.